build.gradle

```diff
@@ -257,7 +257,7 @@ pitest {
   testSourceSets = [sourceSets.integrationTest, sourceSets.test]
   excludedClasses = [
     'uk.gov.hmcts.reform.cwrdapi.config.*',
-    'uk.gov.hmcts.reform.cwrdapi.controller.*',
+    'uk.gov.hmcts.reform.cwrdapi.controllers.*',
```

Comment:
Can we exclude the domain/model classes from pitest? I believe we have not written any logic in model classes(like equals/hashcode or any other method.)

Reply:
We can - however they are fully covered by unit tests now so would be a wasted effort